src/Plugin.php

```diff
@@ -86,6 +90,45 @@ public static function upload_mime_types(array $mimes) {
   }
 
   /**
+   * Removes svg width and height attributes from the content.
+   *
+   * @implements the_content
+   */
+  public static function the_content($content) {
+    $content = preg_replace_callback('@<img [^>]*src=.+?.svg[^>]*>@', function ($matches) {
+      return preg_replace('@((?:width|height)=.+?\s+)@', '', $matches[0]);
+    }, $content);
+    return $content;
+  }
+
+  /**
+   * Removes svg width and height attributes when the image is received via a function.
```

Comment:
1. 👍 
2. E.g. `get_attachment_image()`, `get_post_thumbnail()` etc. anywhere where the metadata for this image is requested. Probably it's better to rewrite the comment and name *when the meta data for the attachment is received* or something.